tests/openwisp2/settings.py

```diff
@@ -71,6 +71,12 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        "NAME": "openwisp_users.password_validation.PasswordReuseValidator",
+    },
+]
```

Comment:
Please include the default validators.

Reply:
This would only create hurdles during development, i.e. we would need to use complex password during development.